doc/rbd/iscsi-initiators.rst

```diff
@@ -12,9 +12,16 @@ Configuring the iSCSI Initiators
 
     .. warning::
 
-        Applications that use SCSI persistent group reservations (PGR) and
-        SCSI 2 based reservations are not supported when exporting a RBD image
-        through more than one iSCSI gateway.
+        The Ceph iSCSI gateway does not support reservations in any scenario.
+        ``tcmu-runner`` does not support reservations.
```

Comment:
Pasting my response to your earlier question on whether the existing statement is misleading:
> It's true that tcmu-runner doesn't have any code to handle persistent reservations -- it can't because that information isn't passed through by the kernel.  But I think with a single gateway persistent reservations could be reasonably handled by the LIO target in the kernel.  Not sure if that actually works or useful -- we haven't tested this aspect in years -- but all in all I wouldn't call that sentence misleading.

It looks like you chose to amend it anyway, why?

Reply:
This was a pure boneheaded move on my part. I was doing a lot of docs things in a row, and I just plain forgot.

Sorry about that.

I put it back the way it was.

Reply:
No problem, LGTM!